packages/terra-site/package.json

```diff
@@ -50,6 +50,7 @@
     "terra-dynamic-grid": "^1.6.1",
     "terra-embedded-content-consumer": "^1.6.1",
     "terra-form": "^1.12.0",
+    "terra-form-select": "^0.x",
```

Comment:
should this be ^0.0.0?

Reply:
This is usually how we reference components before the v1 release.